rero_ils/modules/holdings/api.py

```diff
@@ -655,7 +655,7 @@ def create_holding(
         notes=[], vendor_pid=None, acquisition_status=None,
         acquisition_expected_end_date=None, acquisition_method=None,
         general_retention_policy=None, completeness=None,
-        composite_copy_report=None, issue_binding=None):
+        composite_copy_report=None, issue_binding=None, _masked=False):
```

Comment:
the `_` prefix is presume to be use for 'private' attribute/method in Python.
I think it's better to use `masked` as argument name instead of `_masked`